Cinema system/Cinema system/Services/CinemaService.cs

```diff
@@ -81,5 +83,105 @@ public void DeleteCinema(int cinemaId)
 
             _db.SaveChanges();
         }
+
+        public bool IsHallExists(int cinemaId, int hallId)
```

Comment:
Name starting with "Is..." shouldn't be used as method name. It's more suitable for object properties.
Also something wrong with the grammar here.
Better name would be "CheckHallExists"